src/cli-sdk/src/index.ts

```diff
@@ -47,25 +50,49 @@ const loadCommand = async <T>(
   /* c8 ignore stop */
 }
 
+const isErrorWithCauseObject = (
+  err: unknown,
+): err is ErrorWithCauseObject =>
+  !!err &&
+  typeof err === 'object' &&
+  err instanceof Error &&
+  !!err.cause &&
+  typeof err.cause === 'object' &&
+  'code' in err.cause &&
+  err.cause.code === 'JACKSPEAK' &&
+  'found' in err.cause
+
 const loadVlt = async (cwd: string, argv: string[]) => {
   try {
     return await Config.load(cwd, argv)
   } catch (err) {
-    if (
-      err instanceof Error &&
-      err.cause &&
-      typeof err.cause === 'object' &&
-      'code' in err.cause &&
-      err.cause.code === 'JACKSPEAK' &&
-      'found' in err.cause
-    ) {
-      const { found } = err.cause
+    if (isErrorWithCauseObject(err)) {
+      const { found, path, wanted, name } = err.cause
+      const isConfigFile = typeof path === 'string'
+      const msg =
+        isConfigFile ?
+          `Problem in Config File ${path}`
+        : 'Invalid Option Flag'
+      const validOptions =
+        wanted ? undefined
+        : isConfigFile ? getSortedKeys()
+        : getSortedCliOptions()
+
+      stderr(msg)
+      stderr(err.message)
+      if (name) stderr(indent('Field: ' + name))
+      if (found) stderr(indent('Found: ' + JSON.stringify(found)))
+      if (wanted) stderr(indent('Wanted: ' + wanted))
```

Comment:
If we are in this block it means we have no config so we can't use the same formatting options we use with `util.formatWithOptions`. But we should still use some sort of format function here even if its just `util.format`. This will fix the linting errors.

Reply:
At this point we know it's coming from Jackspeak, and thus `wanted` is always a string, so I fixed it by just checking `if (typeof wanted === 'string')` which future proofs it and also satisfies lint.